tests/forms_tests/field_tests/test_emailfield.py

```diff
@@ -51,3 +51,13 @@ def test_emailfield_min_max_length(self):
         self.assertEqual('alf@foo.com', f.clean('alf@foo.com'))
         with self.assertRaisesMessage(ValidationError, "'Ensure this value has at most 15 characters (it has 20).'"):
             f.clean('alf123456788@foo.com')
+
+    def test_emailfield_strip_on_none_value(self):
+        """EmailField 'correctly' strips None value instead of raising error (Refs #26821)"""
```

Comment:
I think the test is simple and straightforward enough that a docstring isn't needed. We only mention ticket numbers for complex issues where the context of ticket might be helpful in understanding what's going on.